app/presenters/admin_email_list.rb

```diff
@@ -11,6 +9,14 @@ def initialize(list_names)
     @list_names = [list_names].flatten.map(&:to_s)
   end
 
+  def available_open_studios_events
```

Comment:
This was one of the few places we want *all* os events.  So the shim logic ended up moving here bit we focus on the event here, instead of the `key`